common.py

```diff
@@ -52,11 +52,13 @@ class UploadInfo:
     size_bytes: int = None
 
     def from_dict(d):
-        return UploadInfo(**{
-            k: v for k, v in d.items()
-            if k in inspect.signature(UploadInfo).parameters
-        })
-
+        return UploadInfo(
+            **{
+                k: v
+                for k, v in d.items()
+                if k in inspect.signature(UploadInfo).parameters
+            }
+        )
```

Comment:
whoops, my python prettifier is trying to format things differently, I'll get your settings and change it back